orangewidget/utils/signals.py

```diff
@@ -52,18 +52,20 @@ def base_summarize(_) -> PartialSummary:
 """
 
 
-def can_summarize(type_, name):
+def can_summarize(type_, name, explicit):
+    if explicit is not None:
+        return explicit
     if not isinstance(type_, tuple):
         type_ = (type_, )
     instr = f"To silence this warning, set auto_sumarize of '{name}' to False."
     for a_type in type_:
-        try:
-            summarizer = summarize.dispatch(a_type)
-        except TypeError:
-            warnings.warn(f"{a_type.__name__} cannot be summarized. {instr}",
-                          UserWarning)
+        if isinstance(a_type, str):
+            warnings.warn(
+                f"Output is specified with qualified name ({a_type}). "
+                "To enable auto summary, set auto_summarize to True. "
```

Comment:
auto_summarize or auto_summary?

Reply:
Fixed -- here and a few lines above, in the existing code.